examples/nextjs-live-cursors-chat/liveblocks.config.ts

```diff
@@ -0,0 +1,32 @@
+import { createClient } from "@liveblocks/client";
+import { createRoomContext } from "@liveblocks/react";
+
+const client = createClient({
+  publicApiKey: process.env.NEXT_PUBLIC_LIVEBLOCKS_PUBLIC_KEY!,
+});
+
+type Presence = {
+  cursor: {
+    x: number;
+    y: number;
+  } | null;
+  message: string;
+};
+
+type Storage = {};
```

Comment:
Let's add a comment here to illustrate that this is deliberately not using Storage APIs:

```suggestion
// Storage APIs are not used in this example app
type Storage = {};
```